core/templates/pages/exploration-editor-page/modal-templates/state-diff-modal.component.spec.ts

```diff
@@ -112,4 +112,20 @@ describe('State Diff Modal Component', () => {
       expect(component.yamlStrs.rightPane).toBe('Yaml data');
     });
   }));
+
+  it('should evaluate yaml string objects for translation changes', fakeAsync(() => {
```

Comment:
nit: yaml -> YAML (abbreviations are capitalised in general)

Reply:
Done!